Documentation for latest logout changes (#1624)
keycloak/keycloak-documentation: release_notes/topics/19_0_0.adoc

methods must be reviewed.
See link:{upgradingguide_link}[{upgradingguide_name}] for details.

= OIDC Logout changes

> In the previous release, we added support for OIDC logout. This release contains few other fixes and polishing. Few highlights of them are:

Reviewer: ```suggestion
In the previous release, we added support for OIDC logout. This release contains a few other fixes and polishing. The  highlights include:
```

> - Support for `client_id` parameter, which was added in recent draft of OIDC RP-Initiated Logout specification. Due adding this, it is not needed anymore to use `Consent Required` flag of the

Reviewer: ```suggestion
- Support for the `client_id` parameter, which was added in recent draft of the OIDC RP-Initiated Logout specification. As a result, no need exists to use the `Consent Required` flag of the
```

> client to show logout confirmation screen.

Reviewer: ```suggestion
client to show the logout confirmation screen.
```

> - Configuration option `Valid Post Logout Redirect URIs` added to the OIDC client. This is aligned with the OIDC specification and it allows to use different set of redirect URIs for redirect after login and logout.

Reviewer: ```suggestion
- Configuration option `Valid Post Logout Redirect URIs` added to the OIDC client. This change is aligned with the OIDC specification, which allows you to use a different set of redirect URIs for redirect after login and logout.
```

> Value `+` used for `Valid Post Logout Redirect URIs` means that logout will use same set of redirect URIs as specified by the option of `Valid Redirect URIs`. This is also default behaviour when migrating
> from previous version due the backwards compatibility.
> 
> For more details, see link:{adminguide_link}#_oidc-logout[{adminguide_name}].
> 

Reviewer: ```suggestion
Value `+` used for `Valid Post Logout Redirect URIs` means that the logout will use the same set of redirect URIs as specified by the option of `Valid Redirect URIs`. This change also matches the default behavior when migrating
from a previous version due to backwards compatibility.

For more details, see the link:{adminguide_link}#_oidc-logout[{adminguide_name}].

```
